pages/profile/[profileUrl].js

```diff
@@ -0,0 +1,201 @@
+  return {
+    background: {
+      width: "100%"
+    },
+    avatar: {
+      height: theme.spacing(20),
+      width: theme.spacing(20),
+      margin: "0 auto",
+      marginTop: theme.spacing(-11),
+      fontSize: 50,
+      border: `1px solid ${theme.palette.grey[700]}`
+    },
+    avatarWithInfo: {
+      textAlign: "center",
+      width: theme.spacing(40),
+      margin: "0 auto",
+      [theme.breakpoints.up("sm")]: {
+        margin: 0,
+        display: "inline-block",
+        width: "auto"
+      }
+    },
+    memberInfo: {
+      [theme.breakpoints.up("sm")]: {
+        display: "inline-block"
+      },
+      padding: 0
+    },
+    name: {
+      fontWeight: "bold",
+      padding: theme.spacing(1)
+    },
+    subtitle: {
+      color: `${theme.palette.secondary.main}`
+    },
+    content: {
+      paddingTop: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+      color: `${theme.palette.secondary.main}`,
+      fontWeight: "bold"
+    },
+    noPadding: {
+      padding: 0
+    },
+    infoContainer: {
+      [theme.breakpoints.up("sm")]: {
+        display: "flex"
+      }
+    },
+    cardHeadline: {
+      marginTop: theme.spacing(3),
+      marginBottom: theme.spacing(1)
+    },
+    noprofile: {
+      textAlign: "center",
+      padding: theme.spacing(5)
+    }
+  };
+});
+
+export default function ProfilePage({ profile, projects, organizations }) {
+  console.log(profile);
+  return (
+    <WideLayout title={profile ? profile.name + "'s profile" : "Not found"}>
+      {profile ? (
+        <ProfileLayout profile={profile} projects={projects} organizations={organizations} />
+      ) : (
+        <NoProfileFoundLayout />
+      )}
+    </WideLayout>
+  );
+}
+
+ProfilePage.getInitialProps = async ctx => {
+  return {
+    profile: await getProfileByUrlIfExists(ctx.query.profileUrl),
+    organizations: await getOrganizations(ctx.query.profileUrl),
+    projects: await getProjects(ctx.query.profileUrl)
+  };
+};
+
+function ProfileLayout({ profile, projects, organizations }) {
+  if (projects && !projects.length) {
+    projects = [projects];
+  }
+  const classes = useStyles();
+  return (
+    <Container maxWidth="lg" className={classes.noPadding}>
+      <img
+        src={profile.background_image ? profile.background_image : DEFAULT_BACKGROUND_IMAGE}
+        className={classes.background}
+      />
+      <Container className={classes.infoContainer}>
+        <Container className={classes.avatarWithInfo}>
+          <Avatar
+            alt={profile.name}
+            size="large"
+            src={"/images/" + profile.image}
+            className={classes.avatar}
+          />
+          <Typography variant="h5" className={classes.name}>
+            {profile.name}
+          </Typography>
+          <Typography className={classes.subtitle}>{profile.type}</Typography>
+        </Container>
+        <Container className={classes.memberInfo}>
+          {profile.bio ? (
```

Comment:
I think the following is slightly more idomatic:

```jsx
{profile.bio && (
  <>...</>
)}
```

The same comment applies elsewhere in this PR.